src/disco/topo/fd_topo.h

```diff
@@ -209,6 +209,7 @@ typedef struct {
       char  capture[ PATH_MAX ];
       char  genesis[ PATH_MAX ];
       char  identity_key_path[ PATH_MAX ];
+      char  vote_account_path[ PATH_MAX ];
```

Comment:
there's already a consensus.vote_account_path, are you able to use that ?

Reply:
this is to get the value into the tile, see fd_firedancer.c where we copy in from consensus.vote_account_path.